src/flake8_requirements/checker.py

```diff
@@ -560,6 +563,13 @@ def get_pyproject_toml(cls):
         except (IOError, tomllib.TOMLDecodeError) as e:
             LOG.debug("Couldn't load project setup: %s", e)
             return {}
+        except Exception:
+            try:
+                with open(pyproject_config_path, mode="rb") as f:
+                    return toml.loads(f.read())
```

Comment:
Can you use toml>=0.7.1 ? In that version the `load()` API is also supported: https://github.com/uiri/toml/blob/0.7.1/toml.py#L11

Reply:
Ah, this is a good call, more over the TomlDecodeError is actually introduced in 0.10.0. I will fix it now. I have two questions:
1. Do you want me to alter the commit and force push to keep the history robust, or you will squash the PR or it doesn't matter for you?
2. I just realised that the other PR, that is introducing few new known 3rd parties won't be usable in my case, as I still rely on the PY2 version. Do you mind if I add one more commit to update the known 3rd parties here with the latest state of the list on master?

Reply:
> Do you want me to alter the commit and force push to keep the history robust, or you will squash the PR or it doesn't matter for you?

I'm squashing PR commits, so it doesn't matter.

> I just realised that the other PR, that is introducing few new known 3rd parties won't be usable in my case, as I still rely on the PY2 version. Do you mind if I add one more commit to update the known 3rd parties here with the latest state of the list on master?

I will back-port all changes related to 3rd known parties to the legacy python2 branch.

Reply:
Done in f90ecf016492b1ac9ab8e5ad9e23dea88ef17b85. 
Tested again under 2.7.18, 3.10 and 3.11. 
On 2.7 I tested it with toml 0.7.1 and 0.10.2.